### PR TITLE
extract admin group and label translations

### PR DESCRIPTION
**Pull request by fbourigault**
I am targetting this branch, because it's a BC feature.

## Changelog

```markdown
### Added
- Extract admin group and label translations.
```

## Subject

When using JMSTranslationBundle to extract translations, neither the admin group nor the admin label are extracted. Those are now extracted allowing full extraction of the admin strings using the JMSTranslationBundle extractor.

## Todo

- [x] Tests
- [x] ~~Extract FormMapper groups and tabs labels~~
- [x] ~~Docs~~

### CODE REVIEW

**Comment by greg0ire** [comment]
I think you can add tests for this, can't you. Also, is there anything to improve in the docs? Thanks for your contribution!

**Comment by fbourigault** [comment]
I've wrote some asserts into existing tests. For the docs, I didn't find anything about extracting translations.

**Comment by fbourigault** [comment]
FormMapper groups and tabs labels seems not being extracted. I will give a look at this and also work on this.

**Comment by greg0ire** [comment]
I'll review this when it is no longer wip

**Comment by fbourigault** [comment]
I will not handle groups and tabs labels in this PR. It seems to be more complicated than I expected.

**Comment by greg0ire** [comment]
Ok, let's review it then.

File: Tests/Translator/Extractor/JMSTranslatorBundle/AdminExtractorTest.php [hunk 1]
```diff
@@ -77,8 +77,21 @@ public function setUp()
 
         $logger = $this->getMock('Psr\Log\LoggerInterface');
 
-        $this->pool = new Pool($container, '', '');
-        $this->pool->setAdminServiceIds(array('foo_admin', 'bar_admin'));
+        $this->pool = $this->getMockBuilder('Sonata\AdminBundle\Admin\Pool')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
```
Comment (greg0ire) [inline Tests/Translator/Extractor/JMSTranslatorBundle/AdminExtractorTest.php]: Please be at least consistent with your own code, pick one style and stick to it. I think we usually never put semi-colons on their own lines.

Comment (greg0ire) [inline Tests/Translator/Extractor/JMSTranslatorBundle/AdminExtractorTest.php]: Aside from that it's good to go, I'll merge as soon as you fix this tiny problem.

Comment (fbourigault) [inline Tests/Translator/Extractor/JMSTranslatorBundle/AdminExtractorTest.php]: Code style is now fixed!

**Comment by greg0ire** [comment]
Thanks @fbourigault !

**Pull request merged**